todo/src/components/Todos.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Todo from './Todo';
+class Todos extends Component {
+  render() {
+    return (
+      <ul>
+        {props.todos.map(todo => (
+          <Todo
+            value={todo.value}
+            key={todo.id}
+            id={todo.id}
+            completed={todo.completed}
```

Comment:
If you want to use `todo.id`, `todo.completed`, or `todo.value`, you need to make sure that `todo` is an object with the properties.